src/Smartsheet.php

```diff
@@ -138,10 +161,12 @@ function ( $carry, $cell ) use ( $columns, &$cell_index ) {
 	 * @return int|null The id of the column, or null if no column was found.
 	 */
 	public function get_column_index( string $column_title ): ?int {
+		$i = 0;
```

Comment:
I think `return array_search( $column_title, array_column( $this->columns, 'title' ) ) ?? null;` also does the job

Reply:
Changed it, but `??` only works for null, so I had to use `?:` and allow its syntax here.

Reply:
Ah my bad :facepalm:  but then `a ?: b` returns b if a is 0, which can clearly happen with array indexes
Probably better in 2 steps 
```php
$index = array_search( $column_title, array_column( $this->columns, 'title' ) );
return $index !== false ? $index : null;
```